build.gradle

```diff
@@ -353,6 +353,21 @@ task sparkJar(type: ShadowJar) {
     classifier = 'spark'
 }
 
+task distArchive(type: Zip) {
+    dependsOn shadowJar, sparkJar
+
+    baseName = project.name + "-" + project.version
+    destinationDir file("$buildDir")
+    archiveName baseName + ".zip"
+
+    from(shadowJar.archivePath)
+    from(sparkJar.archivePath)
+    from("settings.gradle")
+    from("gatk-launch")
+    from("README.md")
```

Comment:
I can't find a way to get gradle to blow up if one of the files to be included in the archive does not exist -- instead, it happily creates an incomplete archive. @lbergelson do you happen to know of a way to do this?